ironfish/src/workerPool/tasks/decryptNotes.ts

```diff
@@ -26,6 +26,7 @@ export interface DecryptedNote {
 }
 
 export class DecryptNotesRequest extends WorkerMessage {
```

Comment:
Another null value place holder for serialization/deserialization. Open to comments